build/gulpfile.vscode.js

```diff
@@ -424,6 +424,10 @@ function packageTask(platform, arch, sourceFolderName, destinationFolderName, op
 				.pipe(replace('@@PRODNAME@@', product.nameLong))
 				.pipe(replace('@@APPNAME@@', product.applicationName))
 				.pipe(rename('bin/' + product.applicationName)));
+		} else if (platform === 'darwin') {
+			const policyDest = path.join(`${product.nameLong}.app`, 'Contents', 'Resources', 'app', 'policies');
+			result = es.merge(result, gulp.src('.build/policies/darwin/**', { base: '.build/policies/darwin' })
+				.pipe(rename(f => f.dirname = path.join(policyDest, f.dirname))));
```

Comment:
Do it above the call to `electron()` so as to avoid hardcoding paths within the app bundle:

https://github.com/microsoft/vscode/blob/747b90cfcd486731a1295563423b1ee66dd9b613/build/gulpfile.vscode.js#L371-L378

Reply:
Ah yes thanks for the hint. That's much better.  Fixed in https://github.com/microsoft/vscode/pull/241773/commits/7c90a9a6c706ae54fb338614755d84095ebe44e1

<img width="827" alt="Screenshot 2025-03-13 at 11 21 31 AM" src="https://github.com/user-attachments/assets/d1d5e440-4931-4133-9ad6-0d84624ab714" />